src/main/kotlin/com/exactpro/th2/conn/amqp/configuration/Configuration.kt

```diff
@@ -14,15 +14,19 @@
  * limitations under the License.
  */
 
-package com.exactpro.th2.conn.ampq.impl
+package com.exactpro.th2.conn.amqp.configuration
 
 class Configuration(
     val sessionAlias: String,
+    val enableMessageSendingEvent: Boolean = true,
     val drainIntervalMills: Long = 1000L,
-    val rootEventName: String = "ConnTemplate",
+    val rootEventName: String = "ConnAmqp",
```

Comment:
I think it should be nullable and if it is not set we need to use the box name (from box config in CommonFactory)